src/index.ts

```diff
@@ -11,4 +11,6 @@ export {
 export {
   WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT,
 } from './utilities/within-content-context';
+
+export {buildColors as unstableBuildColors} from './utilities/theme/utils';
```

Comment:
let's make it _really_ clear :) 

```suggestion
export {buildColors as UNSTABLE_BuildColors} from './utilities/theme/utils';
```